core/domain/suggestion_services.py

```diff
@@ -368,9 +368,9 @@ def get_reviewable_suggestions(user_id, suggestion_type):
         language_codes = (
             user_review_rights.can_review_translation_for_language_codes)
         return [
-            suggestion for suggestion in all_suggestions if (
-                suggestion.change.language_code in language_codes)
-            ]
+            suggestion for suggestion in all_suggestions if
+            suggestion.change.language_code in language_codes]
```

Comment:
Optional nit: slightly nicer to move the "if" to the front of this line, since it starts a new clause.

Reply:
Done!